release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2069,6 +2069,14 @@ As a result, if you apply a performance profile, all nodes in the cluster reboot
 +
 To revert all nodes in the cluster to the cgroups v2 configuration, you must edit the `Node` resource. For more information, see xref:../nodes/clusters/nodes-cluster-cgroups-2.adoc#nodes-clusters-cgroups-2_nodes-cluster-cgroups-2[Configuring Linux cgroup v2]. You cannot revert the cluster to the cgroups v2 configuration by removing the last performance profile. (link:https://issues.redhat.com/browse/OCPBUGS-16976[*OCPBUGS-16976*])
 
+* There is a known issue with using Azure confidential virtual machines, which is a Technology Preview feature in this release. Configuring a cluster to encrypt the managed disk and the Azure VM Guest State (VMGS) blob with a platform-managed key (PMK) or a customer-managed key (CMK) is unsupported. To avoid this issue, only enable encryption of the VMGS blob by setting the value of the `securityEncryptionType` parameter to `VMGuestStateOnly`. (link:https://issues.redhat.com/browse/OCPBUGS-18379[*OCPBUGS-18379*])
+
+* There is a known issue with using Azure confidential virtual machines, which is a Technology Preview feature in this release. Installing a cluster configured to use this feature sometimes fails because the control plane provisioning process times out after 30 minutes.
++
+If this occurs, you can run the `openshift-install` command with the xref:../installing/installing_azure/installing-azure-user-infra.adoc#installation-azure-user-infra-wait-for-bootstrap_installing-azure-user-infra[`wait-for bootstrap-complete`] and xref:../installing/installing_azure/installing-azure-user-infra.adoc#installation-azure-user-infra-completing_installing-azure-user-infra[`wait-for install-complete`] arguments to complete the installation.
```

Comment:
I don't think this is valid workaround to avoid the issue.
Due to bootstrap timeout issue, bootstrap host is not destroyed even you rerun command `wait-for bootstrap-complete' again, that will cause ingress operator degraded and we could not get successful cluster.

Before OCPBUGS-18488 is fixed,  I think installation with security type `ConfidentiallVM` (whatever the value of `securityEncryptionType` to be set)  will be totally unsupported.

@mresvanis WDYT?

Reply:
I also tested with the `openshift-install wait-for bootstrap-complete` and it indeed did not destroy the bootstrap node. But when issuing an additional `openshift-install create cluster ...` command, everything seems to be working as expected.

Reply:
@mresvanis thanks for adding more info. Checking for my understanding, are your steps as follows?

1. Try to install normally, get timeout
2. Try again with `openshift-install wait-for bootstrap-complete`
3. Try a third time, with just the normal install command (no `wait-for install-complete` required)
4. Successful install

If so, is step 1. even needed? Or does it make sense to start with `wait-for bootstrap-complete` if you want to install with this feature?

Reply:
Ok, thanks. @jinyunma does simply running the `openshift-install create cluster` a second time (with no `wait-for` options) seem okay to you too? 

I pushed a [commit](https://github.com/openshift/openshift-docs/pull/66398/commits/5f0629d8ff97b5a6b3359122de81c2bbfb95695b) to show what this would look like, let me know what you think.

Reply:
@mresvanis I tested to run `openshift-install create cluster` second time, it indeed worked, bootstrap was destroyed this time, and installation was successful finally.

@jeana-redhat the change LGTM.

Reply:
Wonderful, thank you both for getting through all of this to deliver it in time :tada: I am going to squash the commits and get this merged today.